packages/mendel-outlet-manifest/package.json

```diff
@@ -17,6 +17,7 @@
     "babel-plugin-transform-inline-environment-variables": "0.0.2",
     "debug": "^2.6.0",
     "mendel-manifest-uglify": "^3.0.2",
+    "mendel-development": "^3.0.1",
```

Comment:
Not sure about this dependency. What else does Mendel development do? From the name of the package, it is support hard to know.